### PR TITLE
Bug 1259776: Display ban info on user profile

### PR DESCRIPTION
**Pull request by stephaniehobson**
Displays ban info to users with ban permissions instead of super user permissions.

Adds ban date and moderator to user profile page.

### CODE REVIEW

**Comment by escattone** [comment]
Based on this PR, my understanding is that the current logic for determining a user's "access" to banned info is by checking `user.has_perm('users.add_userban')` rather than `user.is_superuser`? If that's the case, it appears that the logic in `kuma/users/view.py::user_detail` should be changed as well?

**Comment by stephaniehobson** [comment]
Ooh, yes, good catch.

**Comment by stephaniehobson** [comment]
Updated :hammer:

File: kuma/users/views.py [hunk 1]
```diff
@@ -352,7 +352,7 @@ def user_detail(request, username):
     """
     detail_user = get_object_or_404(User, username=username)
 
-    if (detail_user.active_ban and not request.user.is_superuser):
+    if (detail_user.active_ban and not request.user.add_userban):
```
Comment (jwhitlock) [inline kuma/users/views.py]: Should be ``request.user.has_perm('users.add_userban')``, not ``request.user.add_userban``.

**Comment by stephaniehobson** [comment]
Updated :aerial_tramway:

**Pull request merged**